src/site/topics/java-vs-python/java-vs-python.rst

```diff
@@ -441,37 +493,48 @@ Functions
                 // Java
                 public class SomeClass {
                     public static void main(String[] args) {
-                        // Call the function
-                        int result = someFunction(1, 2);
+                        // Call the method
+                        int result = someMethod(1, 2);
                         System.out.println(result);
                     }
 
-                    // Declare the Function
-                    static int someFunction(int a, int b) {
+                    // Declare the Method
+                    static int someMethod(int a, int b) {
                         int c = a + b;
                         return c;
                     }
                 }
 
-* We **must** tell the function its return type
-    * ``int`` in this example, because the thing being returned is an int
-    * If we happen to have no return value for our function, then use ``void``
-        * ``static void someOtherFunction( ... ) { ``
 
-* Parameters have their types set
+* In Java, methods must be explicitly told what their return type is
+
+    * ``int`` in this example, because the value being returned is an ``int``
+    * In the case where the method returns no value, the return type is set to ``void``
+
+        * ``static void someOtherMethod( ... ) {``
+
+
+* Parameters have their types included
+
+    * ``int a`` and ``int b`` in the parameter list
+
+* In the above example, the method is ``static``, which means it can act more like a function than a method
 
-* In the above example, the function is ``static``
-    * This is a function that belongs to the class, not an instance of the class
-        * In fact, we could have used this instead ``int result = SomeClass.someFunction(1, 2);``
-    * It's not a function we will call on an instance of some object
+    * This is a method that belongs to the class, not an instance of the class
+    * In other words, it's not a method we will call on an instance of some object
 
-* You will also see that, unlike in the Python example, we can declare the functions after the calling code
+
+* As seen in the above example, the method is defined *after* it is called
+
+    * The method ``someMethod`` is called within ``main``, but it is written below ``main``
+    * This is not required in Java, but is something one could do
+    * In Python, the interpreter needed to know the function existed before it could be referenced
 
 
 Temperature Converter
 ---------------------
 
-.. list-table:: Function to Convert Fahrenheit to Celsius
+.. list-table:: Function/Method to Convert Fahrenheit to Celsius
```

Comment:
Moved from above because it belonged here. Regardless, talked offline.

> My understanding is that they are by definition still methods, despite totally being functuons.
> Nevertheless, I would argue:
> 1. Differentiating function from method in this context is helpful, even if they in fact are technically methods. The spirit of the function vs Java holds well here and is helpful.
> 2. Really splitting hairs.

>Long story short --- for simplicity, functions are the superior word choice here.